tests/PRActionData/TestProcessPr.test_assign.json

```diff
@@ -80,7 +80,7 @@
     },
     {
         "type": "create-comment",
-        "data": "New categories assigned: dqm,db\n\nrvenditti,syuvivida,tjavaid,nothingface0,antoniovagnerini,francescobrivio,saumyaphor4252,perrotta,consuegs you have been requested to review this Pull request/Issue and eventually sign? Thanks"
+        "data": "New categories assigned: dqm,db\n\nantoniovagnerini,consuegs,francescobrivio,nothingface0,perrotta,rvenditti,saumyaphor4252,syuvivida,tjavaid you have been requested to review this Pull request/Issue and eventually sign? Thanks"
```

Comment:
@iarspider , any suggestions? should we update the test file directly here or do you want to auto generate it?

Reply:
@smuzaffar yes, we can update it manually.